media/asciidoctor.css

```diff
@@ -232,7 +232,7 @@ table.pyhltable .linenodiv{background:none!important;padding-right:0!important}
 .quoteblock .quoteblock blockquote{padding:0 0 0 .75em}
 .quoteblock .quoteblock blockquote:before{display:none}
 .verseblock{margin:0 1em 1.25em 1em}
-.verseblock pre{font-family:"Open Sans","DejaVu Sans",sans;font-size:1.15rem;color:rgba(0,0,0,.85);font-weight:300;text-rendering:optimizeLegibility}
+.verseblock pre{font-family:"Open Sans","DejaVu Sans",sans,sans-serif;font-size:1.15rem;color:rgba(0,0,0,.85);font-weight:300;text-rendering:optimizeLegibility}
```

Comment:
I think this needs to be fixed upstream first https://github.com/redhat-developer/vscode-didact/blob/b85864c1cec9945b1d78283cf7c857c8b4a44c5a/media/asciidoctor.css

Reply:
We have a copy of the file. At best we can make a suggestion for the fix in an issue/pr, but they are definitely not under any obligation to fix it.

Reply:
the advantages to provide the fix upstream first are:
- have feedback from the maintainers of the original files
- avoid diverging copy, which allows easier maintenance

it is also fair to provide fix on code that we reuse.

Reply:
I have created https://github.com/darshandsoni/asciidoctor-skins/issues/39 which includes a description of the fix.

Reply:
I am spinning this off here - https://github.com/redhat-developer/vscode-didact/pull/292 - based on feedback I received on https://github.com/darshandsoni/asciidoctor-skins/issues/39